README.md

```diff
@@ -576,6 +576,36 @@ database {
 
 By default batch inserts will be processed in blocks of 2000. To modify this value add the field `insert-batch-size` to the `database` stanza.
 
+Cromwell metadata (paths to job results, start and end times, etc.) grow at a significantly faster rate than operations
+data (running workflows, previous call cache results, etc.). To use a separate database for operations and/or metadata,
```

Comment:
TOL: there is no other information about operations data in the README, so 
1) should there be more info about operations metadata? 
2) maybe you could add a little more information here about it; ex what about the running workflows is returned, or about previous call cache results; Is it the number of each? Their workflow id? Status?